stable_baselines/common/env_checker.py

```diff
@@ -0,0 +1,142 @@
+import warnings
+
+import gym
+from gym import spaces
+import numpy as np
+
+from stable_baselines.common.vec_env import DummyVecEnv, VecCheckNan
+
+
+def check_env(env, warn=True):
```

Comment:
Consider splitting up into separate (module-private) helper methods. The code was clear individually but at 130 lines it's hard to get a quick overview of everything it was doing.

Reply:
Good point